docroot/themes/humsci/humsci_basic/src/scss/utilities/_card-images.scss

```diff
@@ -8,7 +8,7 @@
       }
 
       @include grid-media-min($hb-horizontal-card-breakpoint) {
-        .hb-card--horizontal .hb-card__content {
+        .hb-card--horizontal:not(.hb-card--no-image) .hb-card__content {
```

Comment:
what does this do? I am having trouble figuring out why you need this class when you declared other styles to rule out flex being added to the `[class^="hb-card-image-"] .hb-card--no-image &` class in the card.scss?

Reply:
Normally `.hb-card__content` flexes to fill the remaining space next to the image but when we use the utility classes we had to set a max width on the content area or else the text would overflow the container when the image is really wide. 

If there is no image then the content was still getting a max width and there would be a gap between the right of the content and the edge of the container.